css/main.css

```diff
@@ -325,3 +325,48 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.contact_form_section {
+  height: 100vh;
+  display: flex;
+}
+
+.contact_form_header {
+  max-width: 20vw;
+  margin: 50px 30px 0 10vw;
+}
+
+.form_container {
+  margin: 50px 0 0 0;
+  width: 40vw;
+}
+
+input {
+  border: none;
+  border-bottom: solid 1px lightgray;
+  margin: 5px 0 0 5px;
+}
+
+#first_name{
+  width: 39%; 
+}
+
+#last_name{
+  width: 39%;
+}
+
+#email_address {
+  width: 80%;
+}
+
+textarea {
+  margin: 5px;
+  width: 80%;
+  border: none;
+  background-color: rgb(241, 233, 216);
+}
```

Comment:
- [ ] **[OPTIONAL]** If you added the desired `width` for your form as suggested above, you can just change all of the `width: 80%` to `width: 100%`. So if you ever change the desired `width` on your `.form-container`, it will change in all of its children!

- [ ] **[OPTIONAL]** About `first_name` and `last_name`: Instead of setting very specific widths to these 2 properties, you could have nested these 2 `inputs` inside a `div` and change the `div` display to `flex` and add a `gap`:

```css
#div_name {
  width: 100%; /* The .form-container already gives you the desired size */
  display: flex;
  gap: 2%; 
}

#first_name,
#last_name {
  width: 50%;
}
```